app/webpacker/styles/primary-nav.scss

```diff
@@ -1,20 +1,29 @@
+$primaryNavHeight: 2.5rem;
+
 .primary-nav-container {
+  background-color: govuk-colour("light-grey");
+  border-bottom: 1px $govuk-border-colour solid;
+}
+
+.primary-nav {
   display: flex;
   flex-direction: row;
-  background-color: govuk-colour("light-grey");
-  height: 2.5rem;
-  padding-left: 4px;
+  height: $primaryNavHeight;
 }
 
 .nav-item {
-  @include govuk-typography-responsive($size: 27);
-  padding-top: 0.25rem;
+  @include govuk-typography-responsive($size: 19);
+
+  &:not(:last-child) {
+    margin-right: govuk-spacing(8);
+  }
 
   &.nav-item__selected {
     border-bottom: solid govuk-colour("blue") 4px;
   }
 
   a {
     text-decoration: none;
+    line-height: $primaryNavHeight;
```

Comment:
Tip for the future @ethanmills - this is a neat trick to vertically align text in navigation and avoid the `padding-top: 0.25rem` that was there before (which breaks when the text size changes)

Reply:
Thanks!